CONTRIBUTING.md

```diff
@@ -42,9 +44,9 @@ Tabby can be run through `cargo` in much the same manner as docker:
 cargo run serve --model TabbyML/StarCoder-1B
 ```
 
-This will run Tabby locally on CPU, which is not optimal for performance. Depending on your GPU and its compatibility, you may be able to run Tabby with GPU acceleration. Please make sure you have CUDA or ROCm installed, for Nvidia or AMD graphics cards respectively.
+This will run Tabby locally on CPU, which is not optimal for performance. Depending on your GPU and its compatibility, you may be able to run Tabby with GPU acceleration. Please make sure you have CUDA or ROCm installed, for Nvidia or AMD graphics cards respectively. No extra library installation is necessary for Apple silicon (M1/M2) using Metal.
 
-To run Tabby locally with CUDA (Nvidia):
+To run Tabby locally with CUDA (NVIDIA):
 
 ```
 cargo run --release --features cuda serve --model TabbyML/StarCoder-1B --device cuda
```

Comment:
```suggestion
cargo run --features cuda serve --model TabbyML/StarCoder-1B --device cuda
```

For day to day development, we don't wanna `--release` as it slows down compiling

Reply:
I just ran the same test in release and debug mode, both with GPU enabled. Took over 20 seconds to fill in hello world in python (in a chat session) on debug mode, and under half a second in release mode.

Reply:
5 seconds to complete it in an IDE on debug, instantaneous on release